collectors/python.d.plugin/rabbitmq/README.md

```diff
@@ -113,4 +113,23 @@ socket:
 
 ---
 
+### Per-Queue Chart configuration
 
+RabbitMQ users with the "monitoring" tag cannot see all queue data. You'll need a user with read permissions. 
+To create a dedicated user for netdata:
+
+```bash
+rabbitmqctl add_user netdata ChangeThisSuperSecretPassword
+rabbitmqctl set_permissions netdata "^$" "^$" ".*"
+```
```

Comment:
```suggestion
```

See [set_permissions](https://www.rabbitmq.com/rabbitmqctl.8.html#set_permissions) for details.

```